### ISSUE TITLE
dca 5.1ch in BD m2ts use --audio-stream stereo failed

### ISSUE THREAD

**Issue by aliofwind** [rigaya/NVEnc#251]
```
G:\>nvencc64 -i 00002.m2ts -u quality --vbrhq 5000 --audio-codec 1?aac --audio-bitrate 1?192 --audio-stream 1?stereo -o 00002.mp4
--------------------------------------------------------------------------------
00002.mp4
--------------------------------------------------------------------------------
[Parsed_pan_0 @ 0000024b1cbb3240] Invalid channel layout '2ch'
[AVFilterGraph @ 0000024b07550380] Error initializing filter 'pan' with args '2ch|c0=c0|c1=c1'
avout: Failed to parse filter description: Invalid argument: "(null)"
failed to initialize file reader(s).
Failed to open output file: "00002.mp4"

```
change "stereo" to "2.1" or other channel, it working

```
G:\>nvencc64 -i 00002.m2ts -u quality --vbrhq 5000 --audio-codec 1?aac --audio-bitrate 1?192 --audio-stream 1?2.1 -o 00002.mp4
--------------------------------------------------------------------------------
00002.mp4
--------------------------------------------------------------------------------
NVEncC (x64) 5.11 (r1608) by rigaya, Jul 15 2020 13:03:22 (VC 1916/Win/avx2)
OS Version     Windows 10 x64 (17134)
CPU            AMD Ryzen 5 1600 Six-Core Processor (6C/12T)
GPU            #0: GeForce GTX 1050 Ti (768 cores, 1392 MHz)[PCIe3x16][446.14]
NVENC / CUDA   NVENC API 10.0, CUDA 11.0, schedule mode: auto
Input Buffers  CUDA, 20 frames
Input Info     avcuvid: H.264/AVC, 1920x1080, 24000/1001 fps
Vpp Filters    copyDtoD
Output Info    H.264/AVC high @ Level auto
               1920x1080p 1:1 23.976fps (24000/1001fps)
               avwriter: h264, #1:dca/5.1(side):2.1 -> aac/2.1/192kbps => mp4
Encoder Preset quality
Rate Control   VBR
Multipass      2pass-full
Bitrate        5000 kbps (Max: 24000 kbps)
Target Quality auto
Initial QP     I:20  P:23  B:25
VBV buf size   auto
Lookahead      off
GOP length     240 frames
B frames       3 frames [ref mode: disabled]
Ref frames     3 frames
AQ             off
Others         mv:auto cabac deblock adapt-transform:auto bdirect:auto

encoded 1580 frames, 103.08 fps, 4599.85 kbps, 36.14 MB
encode time 0:00:15, CPU: 8.0%, GPU: 4.2%, VE: 81.9%, VD: 32.6%, GPUClock: 1622MHz, VEClock: 1453MHz
frame type IDR    7
frame type I      7,  avgQP  22.71,  total size   0.90 MB
frame type P    395,  avgQP  24.19,  total size  18.05 MB
frame type B   1178,  avgQP  29.70,  total size  17.18 MB
```

**Comment by rigaya** [comment]
Thank you for reporting, should be fixed in [NVEnc 5.12](https://github.com/rigaya/NVEnc/releases/tag/5.12).

**Comment by rigaya** [comment]
I'll close this issue, as the topic should have been already fixed. Please reopen if you have any related issues.
